tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "extends": "@vue/tsconfig/tsconfig.web.json",
+  "compilerOptions": {
+    "lib": ["ES2022", "DOM", "DOM.Iterable"],
+    "baseUrl": ".",
+    "allowJs": true,
+    "strict": false,
+    "alwaysStrict": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noImplicitAny": true,
+    "strictBindCallApply": true,
+    "strictFunctionTypes": true,
+    "allowUnreachableCode": false,
+    "noImplicitOverride": true,
+    "paths": {
+      "@/*": ["./src/*"]
+    }
+  },
+  "include": ["src/**/*", "src/**/*.vue"],
```

Comment:
You might be missing `vite`'s `env.d.ts`, check a `create-vue` template.

Reply:
Didn't see any
https://github.com/vitejs/vite/blob/main/packages/create-vite/template-vue-ts/tsconfig.json

Maybe on `tsconfig.node.json`? as in [here](https://github.com/vitejs/vite/blob/main/packages/create-vite/template-vue-ts/tsconfig.node.json)

Reply:
Some files are generated by code. Do `pnpm create vue@3` and check it out.